plugins/inputs/aos/aos_streaming/aosstream.pb.go

```diff
@@ -1,11 +1,11 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: aosstream.proto
+// source: aosstream-enduro.proto
```

Comment:
Is this manual or generated? Might be good to leave code names out.

Reply:
yeah it's auto generated, but it's just a comment - I'll take it out.